File: src/main/java/ac/dnd/server/common/propeties/PropertiesSourceConfig.java
package ac.dnd.server.common.propeties;

import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Configuration
@ConfigurationPropertiesScan(basePackages = "ac.dnd.server.common.propeties")
@EnableConfigurationProperties
public class PropertiesSourceConfig {
}

File: src/main/kotlin/ac/dnd/server/common/propeties/PropertiesSourceConfig.kt
package ac.dnd.server.common.propeties

import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.context.annotation.Configuration

@Configuration
@ConfigurationPropertiesScan(basePackages = ["ac.dnd.server.common.propeties"])
class PropertiesSourceConfig